src/borg/archiver.py

```diff
@@ -2699,6 +2703,8 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                               help='do not store ctime into archive')
         fs_group.add_argument('--ignore-inode', dest='ignore_inode', action='store_true',
                               help='ignore inode data in the file metadata cache used to detect unchanged files.')
+        fs_group.add_argument('--cache-mtime', dest='cache_mtime', action='store_true',
+                              help='use mtime instead of ctime in the file metadata cache to detect unchanged files.')
```

Comment:
hmm, in case there are other ideas in future, shall this rather work like:

--cache-timestamp='c' (ctime, default), 'm' (mtime)

or with long names: 'ctime', 'mtime'

Reply:
Perhaps use an environment variable as is done for other rarely-if-ever modified settings.